add-new-exercise.ps1

```diff
@@ -0,0 +1,98 @@
+<#
+.SYNOPSIS
+    Add a new exercise.
+.DESCRIPTION
+    Add the files need to add a new exercise.
+.PARAMETER Exercise
+    The slug of the exercise to add.
+.PARAMETER Topics
+    The topics of the exercise (optional).
+.PARAMETER Core
+    Indicates if the exercise is a core exercise (optional).
+.PARAMETER Difficulty
+    The difficulty of the exercise on a scale from 1 to 10 (optional).
+.PARAMETER UnlockedBy
+    The slug of the exercise that unlocks exercise (optional).
+.EXAMPLE
+    The example below will add the "acronym" exercise
+    PS C:\> ./add-new-exercise.ps1 acronym
+.EXAMPLE
+    The example below will add the "acronym" exercise as a core exercise
+    PS C:\> ./add-new-exercise.ps1 acronym -Core
+.EXAMPLE
+    The example below will add the "acronym" exercise as a core exercise, with
+    two topics, a specified difficulty and being unlocked by "two-fer"
+    PS C:\> ./add-new-exercise.ps1 acronym -Core -Topics strings -Difficulty 3 -UnlockedBy two-fer
```

Comment:
I only see one topic here, but the description above says there are two topics

Reply:
I'll fix this!